core/Plugin/Report.php

```diff
@@ -680,6 +680,16 @@ public function getCategory()
     }
 
     /**
+     * Get the translation key of the category the report belongs to.
+     * @return string
+     * @ignore
+     */
+    public function getCategoryKey()
+    {
+        return $this->category;
+    }
```

Comment:
BTW: There is a method that can give you the translation key for a translated string. Maybe it would be easier to use that function instead of adding this one. I currently don't have the project open but can look the method name up if you want.